flink-connector-kudu/src/main/java/org/apache/flink/connectors/kudu/table/utils/KuduTableUtils.java

```diff
@@ -78,11 +79,20 @@ public static KuduTableInfo createTableInfo(String tableName, TableSchema schema
             List<String> keyColumns = getPrimaryKeyColumns(props, schema);
             ColumnSchemasFactory schemasFactory = () -> toKuduConnectorColumns(columns, keyColumns);
             List<String> hashColumns = getHashColumns(props);
-            int replicas = Optional.ofNullable(props.get(KuduTableFactory.KUDU_REPLICAS)).map(Integer::parseInt).orElse(1);
+            int replicas = Optional.ofNullable(props.get(KuduTableFactory.KUDU_REPLICAS)).map(Integer::parseInt).orElse(3);
```

Comment:
why are we changing the default number of replicas from 1 to 3 ?

Reply:
because the number of copies of the previous version was 3.